libs/template/templates/default/template/{{.project_name}}/resources/{{.project_name_short}}_etl.pipeline.yml.tmpl

```diff
@@ -28,12 +28,11 @@ resources:
 
       libraries:
         - glob:
-            include: transformations/**
+            include: ../src/{{.project_name_short}}_etl/transformations/**
 
-{{- if eq .include_python "yes"}}
       environment:
         dependencies:
-          # By default we include the .whl file generated for the project's pyproject.toml
-          # in the root of this project. Any other dependencies can be added there.
-          - ../../dist/*.whl
-{{- end}}
+          # We include every dependency defined by pyproject.toml by defining an editable environment
+          # that points to the folder where pyproject.toml is deployed.
+          {{- /* We avoid a relative path here to work around https://github.com/databricks/cli/issues/3674 */}}
+          - --editable ${workspace.file_path}
```

Comment:
N.B., this is a workaround for https://github.com/databricks/cli/issues/3674; `-e ..` doesn't work in the current CLI.